src/App.vue

```diff
@@ -93,7 +93,7 @@ export default class App extends Vue {
     .logo .nimiq-logo {
         height: 4rem;
         width: 4rem;
-        margin-right: 2rem;
+        margin-right: 1rem;
```

Comment:
The styleguide says, the gap between the signet and the NIMIQ wordmark should be 0.382 times the signet width. So if the signet is 4rem wide, the margin should be `0.382 * 4rem = 1.528rem`.

Reply:
Yes, you're right. Actually the logo text fixes were a quick fix to roughly match an SVG logo Julian sent me.
The logo label does now perfectly match that svg.